### ISSUE TITLE
Disqus JS needs to escape characters.

### ISSUE THREAD

**Issue by mikestecker** [khalwat/disqus#7]
I have a client who put in a entry title that includes a `'` character which causes the javascript error `Uncaught SyntaxError: Unexpected identifier` when outputting `disqus_title`.

**Comment by mikestecker** [comment]
My current workaround is to escape the entry title before sending it to the plugin, but it would be nice to not have to do this.

``` % set disqusId = entry.postDate.getTimestamp~'_'~entry.id %
{% set disqusTitle = entry.title|e('js') %}
{% set disqusUrl = entry.url|e('js') %}
{{ disqusEmbed(disqusId, disqusTitle, disqusUrl) }}
```

**Comment by khalwat** [comment]
This is addressed in 1.0.2:

https://github.com/nystudio107/disqus
